File: src/app/src/main/java/com/onscreensync/tvapp/datamodels/SignalrReceivedMessage.java
package com.onscreensync.tvapp.datamodels;

import com.google.gson.annotations.SerializedName;

public class SignalrReceivedMessage {
    @SerializedName("deviceId")
    private String deviceId;

    @SerializedName("tenantId")
    private String tenantId;

    @SerializedName("messageType")
    private String messageType;

    @SerializedName("messageData")
    private String messageData;

    @SerializedName("messageStatus")
    private String messageStatus;

    public String getMessageType() {
        return messageType;
    }

    public String getMessageData() {
        return messageData;
    }

    public String getDeviceId() {
        return deviceId;
    }

    public String getTenantId() {
        return tenantId;
    }
}

File: src/app/src/main/java/com/onscreensync/tvapp/datamodels/SignalrReceivedMessage.kt
package com.onscreensync.tvapp.datamodels

import com.google.gson.annotations.SerializedName

data class SignalrReceivedMessage(
    @SerializedName("deviceId")
    val deviceId: String?,
    @SerializedName("tenantId")
    val tenantId: String?,
    @SerializedName("messageType")
    val messageType: String?,
    @SerializedName("messageData")
    val messageData: String?,
    @SerializedName("messageStatus")
    val messageStatus: String?
)
